src/pages/AddStock.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+
+const AddFavoriteStock = (): JSX.Element => {
+  const [stockSymbol, setStockSymbol] = useState('');
+  const [stocks, setStocks] = useState([]);
```

Comment:
Here as well, please take a look at the link. Else, feel free to reach out I would happily walk you throw about the type.